.github/actions/core-cicd/api-limits-check/Readme.md

```diff
@@ -24,7 +24,7 @@ on:
 
 jobs:
   check-rate-limits:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Suggest updating this to reflect how you've implemented with fallback, or even with no fallback at all

```suggestion
    runs-on: ubuntu-${{ vars.UBUNTU_RUNNER_VERSION || '22.04' }}
```